### ISSUE TITLE
A single length value for tts:fontSize specifies both height and width.

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/ttml2#548]
One of two points arising from w3c/ttml1#301

The language in tts:fontSize:

> If a single `<length>` value is specified, then this length applies equally to the horizontal and vertical size of a glyph's EM square; if two `<length>` values are specified, then the first expresses the horizontal size and the second expresses vertical size.

is not clear enough. What it means is that the value _specifies_ or _defines_ both the horizontal and vertical size - the words "applies" and "expresses" are used instead in the same paragraph to mean the same thing, and neither is quite right.

Suggest rewording to:

_If a single `<length>` value is specified, then this length defines equally both the horizontal and vertical size of a glyph's EM square; if two `<length>` values are specified, then the first defines the horizontal size and the second defines the vertical size._

I would potentially be happy with other wording as long as the same is used in both and it is clear that the specified value is setting the em square size.

**Comment by skynavga** [comment]
@palemieux by adding "blocks CR" label, does this mean you object to going to CR for the resolution of an editorial issue?

**Comment by palemieux** [comment]
@skynavga This likely impacts w3c/ttml1#301, which blocks TTML1 CR. Any obstacle to closing this issue?

**Comment by skynavga** [comment]
@palemieux no obstacle, but TTML2 is not dependent on TTML1 CR, so this should not block TTML2 CR in any case

**Comment by nigelmegitt** [comment]
@palemieux I'm not sure which issue you want to close? Anyway, this issue here can be closed by merging the pull request I just opened, which I believe satisfies the resolution made yesterday. You can then port that back to TTML1 if that will help resolve w3c/ttml1#301, or it can be raised as a separate issue on TTML1 to match this one. I don't mind either way.
